### PR TITLE
#116671299 Fix side navigation bar in admin dashboard

### PR DESCRIPTION
**Pull request by johnkariuki**
#### What does this PR do?

Fix side navigation bar in admin dashboard
#### Description of Task to be completed?

Display the sidebar on the admin dashboard
#### How should this be manually tested?
1. Log in as an administrator,
2. Navigate to the admin dashboard and see sidebar
#### Any background context you want to provide?

Minification of the stylesheets resulted in some errors in main.css.
Upon fixing them, I upset the balance on the admin dashboard page
#### What are the relevant pivotal tracker stories?
#116671299
#### Screenshots (if appropriate)

screen shot 2016-03-31 at 20 39 03
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)

### CODE REVIEW

File: resources/views/dashboard/master.blade.php [hunk 1]
```diff
@@ -56,6 +38,12 @@
             $('select').material_select();
           });
         </script>
+
+        <script src = "{!! load_asset('/js/jquery.min.js') !!}"></script>
+        <!-- Materialize Compiled and minified JavaScript -->
+        <script src = "{!! load_asset('/js/materialize.min.js') !!}"></script>
+        <script src = "{!! load_asset('/library/sweetalert/sweetalert.min.js') !!}"></script>
+        <script src = "{!! load_asset('/js/scripts.js') !!}"></script>
         <script src = "{!! load_asset('/js/dashboard.js') !!}"></script>
```
Comment (unicodeveloper) [inline resources/views/dashboard/master.blade.php]: **Code Quality - Nitpick**

Why do we still have `dashboard.js` referenced on its own?..Shouldn't it be minified into `scripts.js` ?

Comment (johnkariuki) [inline resources/views/dashboard/master.blade.php]: My reasoning was that `dashboard.js` is only needed in the admin dashboard so no need to merge and minify it. 

@unicodeveloper Let me do that just now

Comment (johnkariuki) [inline resources/views/dashboard/master.blade.php]: i had actually added `dashboard.js` into the minified `script.js` but had failed to remove it from the admin dashboard master page.

@unicodeveloper This has now been fixed.

**Comment by unicodeveloper** [comment]
I just looked into the folders and I can't see audioplayer and main sass files. Does that mean we don't actually use those sass files? Are we still editing the css files like you did here @andela-jkariuki ?

**Comment by johnkariuki** [comment]
sass files had been ignored by default but I have changed that @unicodeveloper

**Comment by unicodeveloper** [comment]
Where you playing April fool's day joke with me by not un-ignoring it from the .gitignore file before? :thought_balloon: :bow: :bowtie: @andela-jkariuki 

I'll merge this now.

**Pull request merged**